config/core/test-configs.yaml

```diff
@@ -1398,6 +1398,11 @@ device_types:
       - blocklist: *allmodconfig_filter
       - blocklist: {kernel: ['v3.', 'v4.4', 'v4.9']}
 
+  sc7180-trogdor-lazor-limozeen:
```

Comment:
Please send upstream in LAVA: https://git.lavasoftware.org/lava/lava/-/tree/master/etc/dispatcher-config/device-types

Reply:
Requested fork access. Will send as soon as they'll give it.

Reply:
@gctucker https://git.lavasoftware.org/lava/lava/-/merge_requests/1612

Reply:
@gctucker I had to create a new MR, which was almost instantly merged.

https://git.lavasoftware.org/lava/lava/-/merge_requests/1619